src/QsCompiler/CompilationManager/ProjectManager.cs

```diff
@@ -1162,6 +1175,12 @@ editCollection switch
                 this.log?.Invoke("Failed to format document. Formatter may be unavailable.", MessageType.Info);
             }
 
+            // send telemetry if telemetry is enabled
+            var telemetryProps = new Dictionary<string, string?>();
+            telemetryProps["quantumSdkVersion"] = manager?.BuildProperties.SdkVersion?.ToString();
```

Comment:
This might not be needed. If the Extension version is supposed to be the same as QDK version, then we already get the version for every event emitted (every event includes the Extension version).

Reply:
Same command as above; this is the sdk version of the project rather than the extension.